### PR TITLE
Update front page documentation

### PR DESCRIPTION
**Pull request by saarasat**
# Description

A+ front page has undergone some changes, this updates
the documentation to reflect those changes.

Related PR: apluslms/a-plus#560

# Testing

Outlook has been tested. The replaced image was not used anywhere else in the repository.

# Have you updated the README or other relevant documentation?

This is updating the relevant documentation

# Is it [Done](https://wiki.aalto.fi/display/EDIT/Definition+of+Done)?

- [ ] I (developer) have created unit tests and the tests pass
- [ ] I (developer) have created functional tests (Selenium tests) if applicable
- [x] I (developer) have tested the changes manually
- [ ] Reviewer has finished the code review
- [ ] After the review, the developer has made changes accordingly
- [ ] Customer/Teacher has accepted the implementation of the feature

### CODE REVIEW

**Review by raphendyr — APPROVED** [review]

File: introduction/0X_gallery.rst [hunk 1]
```diff
@@ -7,15 +7,18 @@ with A+.
...
-.. figure:: /images/gallery/aplus-front-page.jpg
-
-The above image shows a screenshot of the front page of `A+ at Aalto University
-<https://plus.cs.aalto.fi/>`_ from Summer 2019. The front page shows at most
-eight recent courses. Each course has optional logo image, name, course
-code and teaching time. All courses have a green "Aalto" tag to indicate that
-the course is for Aalto University students. The Ohjelmointistudio 2 course
-has also a blue "MOOC" tag to indicate the course is also a massive open online
-course. Sections `7.1 Settings a course on production servers
+.. figure:: /images/gallery/aplus-front-page.png
```
Comment (mtthwhggns) [inline introduction/0X_gallery.rst:10]: Can you add an `alt` attribute to the image?

File: introduction/0X_gallery.rst [hunk 2]
```diff
@@ -7,15 +7,18 @@ with A+.
...
+
+The above image shows an example screenshot of the front page of `A+ at Aalto University
+<https://plus.cs.aalto.fi/>`_. The front page has two sections. The above sections shows
```
Comment (mtthwhggns) [inline introduction/0X_gallery.rst]:
```suggestion
<https://plus.cs.aalto.fi/>`_. The front page has two sections. The upper sections shows 
```

I think upper and lower makes sense, unless it's above or below something else.

File: introduction/0X_gallery.rst [hunk 3]
```diff
@@ -7,15 +7,18 @@ with A+.
...
-eight recent courses. Each course has optional logo image, name, course
-code and teaching time. All courses have a green "Aalto" tag to indicate that
-the course is for Aalto University students. The Ohjelmointistudio 2 course
-has also a blue "MOOC" tag to indicate the course is also a massive open online
-course. Sections `7.1 Settings a course on production servers
+.. figure:: /images/gallery/aplus-front-page.png
+
+The above image shows an example screenshot of the front page of `A+ at Aalto University
+<https://plus.cs.aalto.fi/>`_. The front page has two sections. The above sections shows 
+all course instances, that the logged-in user has currently enrolled in, is teaching
```
Comment (mtthwhggns) [inline introduction/0X_gallery.rst]:
```suggestion
course instances which the logged-in user is currently; enrolled in, teaching 
```

File: introduction/0X_gallery.rst [hunk 4]
```diff
@@ -7,15 +7,18 @@ with A+.
...
-code and teaching time. All courses have a green "Aalto" tag to indicate that
-the course is for Aalto University students. The Ohjelmointistudio 2 course
-has also a blue "MOOC" tag to indicate the course is also a massive open online
-course. Sections `7.1 Settings a course on production servers
+.. figure:: /images/gallery/aplus-front-page.png
+
+The above image shows an example screenshot of the front page of `A+ at Aalto University
+<https://plus.cs.aalto.fi/>`_. The front page has two sections. The above sections shows 
+all course instances, that the logged-in user has currently enrolled in, is teaching 
+or assisting. The bottom-section shows all ongoing and recently ended courses.
```
Comment (mtthwhggns) [inline introduction/0X_gallery.rst]:
```suggestion
or assisting. The lower section shows all ongoing and recently ended courses.
```

File: introduction/0X_gallery.rst [hunk 5]
```diff
@@ -7,15 +7,18 @@ with A+.
...
+
+Each course has optional logo image, name, instance name, course code and teaching time.
```
Comment (mtthwhggns) [inline introduction/0X_gallery.rst]:
```suggestion
Each course has optional image, name, instance name, course code and teaching schedule. 
```
The image isn't really a logo.

File: introduction/0X_gallery.rst [hunk 6]
```diff
@@ -7,15 +7,18 @@ with A+.
...
+.. figure:: /images/gallery/aplus-front-page.png
+
+The above image shows an example screenshot of the front page of `A+ at Aalto University
+<https://plus.cs.aalto.fi/>`_. The front page has two sections. The above sections shows 
+all course instances, that the logged-in user has currently enrolled in, is teaching 
+or assisting. The bottom-section shows all ongoing and recently ended courses.
+
+Each course has optional logo image, name, instance name, course code and teaching time. 
+All courses have a green "Aalto" tag to indicate that the course is for Aalto University students. 
+The Datastructures and Algorithms -course has also a blue "MOOC" tag to indicate
```
Comment (mtthwhggns) [inline introduction/0X_gallery.rst]:
```suggestion
The "Data Structures and Algorithms" -course has also a blue "MOOC" tag to indicate 
```

File: introduction/0X_gallery.rst [hunk 7]
```diff
@@ -7,15 +7,18 @@ with A+.
...
+
+The above image shows an example screenshot of the front page of `A+ at Aalto University
+<https://plus.cs.aalto.fi/>`_. The front page has two sections. The above sections shows 
+all course instances, that the logged-in user has currently enrolled in, is teaching 
+or assisting. The bottom-section shows all ongoing and recently ended courses.
+
+Each course has optional logo image, name, instance name, course code and teaching time. 
+All courses have a green "Aalto" tag to indicate that the course is for Aalto University students. 
+The Datastructures and Algorithms -course has also a blue "MOOC" tag to indicate 
+the course is also a massive open online course.
```
Comment (mtthwhggns) [inline introduction/0X_gallery.rst]:
```suggestion
that the course is also a massive open online course. 
```

File: introduction/0X_gallery.rst [hunk 8]
```diff
@@ -7,15 +7,18 @@ with A+.
...
-The above image shows a screenshot of the front page of `A+ at Aalto University
-<https://plus.cs.aalto.fi/>`_ from Summer 2019. The front page shows at most
-eight recent courses. Each course has optional logo image, name, course
-code and teaching time. All courses have a green "Aalto" tag to indicate that
-the course is for Aalto University students. The Ohjelmointistudio 2 course
-has also a blue "MOOC" tag to indicate the course is also a massive open online
-course. Sections `7.1 Settings a course on production servers
+.. figure:: /images/gallery/aplus-front-page.png
+
+The above image shows an example screenshot of the front page of `A+ at Aalto University
```
Comment (mtthwhggns) [inline introduction/0X_gallery.rst]:
```suggestion
The above image shows an example of the front page of `A+ at Aalto University
```

The image is a screenshot, the screenshot shows an example.

**Review by mtthwhggns — COMMENTED** [review]
I went through this and made a few suggestions about wording that could be clearer/smoother.

**Comment by saarasat** [comment]
> I went through this and made a few suggestions about wording that could be clearer/smoother.

I made these minor changes

File: introduction/0X_gallery.rst [hunk 9]
```diff
@@ -7,15 +7,19 @@ with A+.
...
+   :alt: Front page of A+
+
+The above image shows an example of the front page of `A+ at Aalto University
+<https://plus.cs.aalto.fi/>`_. The front page has two sections. The upper section shows
+course instances which the logged-in user is currently; enrolled in, teaching
+or assisting. The lower section shows all ongoing and recently ended courses.
+
+Each course has optional image, name, instance name, course code and teaching schedule. 
+All courses have a green "Aalto" tag to indicate that the course is for Aalto University students. 
+"The Data Structures and Algorithms" -course has also a blue "MOOC" tag to indicate
```
Comment (mtthwhggns) [inline introduction/0X_gallery.rst]:
```suggestion
The "Data Structures and Algorithms" -course has also a blue "MOOC" tag to indicate
```

**Review by mtthwhggns — APPROVED** [review]

File: introduction/0X_gallery.rst [hunk 10]
```diff
@@ -7,15 +7,19 @@ with A+.
...
+course instances which the logged-in user is currently; enrolled in, teaching
+or assisting. The lower section shows all ongoing and recently ended courses.
+
+Each course has optional image, name, instance name, course code and teaching schedule. 
+All courses have a green "Aalto" tag to indicate that the course is for Aalto University students. 
+The "Data Structures and Algorithms" -course has also a blue "MOOC" tag to indicate
+that the course is also a massive open online course. 
+Sections `7.1 Settings a course on production servers
 <../m07_admin/01_setup>`_
 and `7.2 Course settings <../m07_admin/02_settings>`_ describe these in detail.
```
Comment (jaguarfi) [inline introduction/0X_gallery.rst]: Now, that you are updating this specific page. Could you fix the internal references?
```suggestion
and :doc:`7.2 Course settings</admin/02_settings>` describe these in detail.
```

**Review by jaguarfi — COMMENTED** [review]
Everything looks fine. Maybe you can fix the references in this page, and then squash all your commits.

File: introduction/0X_gallery.rst (hunk 10 above)
Comment (saarasat) [reply introduction/0X_gallery.rst]: Done

**Review by jaguarfi — APPROVED** [review]
Looks good.

**Pull request merged**